openapi.yaml

```diff
@@ -106,6 +106,18 @@ tags:
     externalDocs:
       description: Find out more
       url: https://pages.nist.gov/OSCAL
+  - name: OSCAL Plan of Action and Milestones
+    externalDocs:
+      description: Find out more
+      url: https://pages.nist.gov/OSCAL/concepts/layer/assessment/poam/
```

Comment:
Is there a reason this points to a different url path than above? For example:
```
 - name: OSCAL Catalog
    externalDocs:
      description: Find out more
      url: https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/
```

Reply:
https://pages.nist.gov/OSCAL/documentation/schema/implementation-layer/component/ goes to https://pages.nist.gov/OSCAL/concepts/layer/implementation/component-definition/

I am asking if you can either make the new `url` match how we did it in the past, or if we should update the old ones to match what they actually link to.

Reply:
Oh, I see. I think we should update the old `urls` to match what they actually link to.